apps/airview-demo/src/views/document-view/external-content.js

```diff
@@ -20,5 +22,22 @@ export function ExternalContent({ metadata }) {
     getData();
   }, [setData, metadata]);
 
-  return <MDEditor.Markdown source={data} />;
+  const components = {
+    img: ({ src, alt }) => {
+      if (!src.startsWith("https:")) {
+        const relativeUrl = src.replace(/^\//, "");
+
+        const path = new URL(
+          relativeUrl,
+          `${githubBaseUri}${metadata.external_path}`
```

Comment:
If I understand what is happening, we are using `githubBaseUri` to construct the relative link but not using its value when sent up to the API, i.e just sending the relative path of the image but need a base url because the `URL` contructor needs it? If so, can we use a placeholder url, e.g. `example.com` to save confusion?

Reply:
Yes, you are right, it only serves the purpose to get the path of the image, so it can be anything. I have changed it to what you suggested to avoid confusion.